vlp/api/helpers.py

```diff
@@ -4,19 +4,25 @@
 from moviepy.editor import VideoFileClip
 from scenedetect import open_video, SceneManager
 from scenedetect.detectors import ContentDetector
-from .models import URL
+from .models import URL, Query
 from django.db import transaction
 from googleapiclient.discovery import build
+from datetime import datetime
+
 
 # Definining download directory
 download_directory = os.path.join(BASE_DIR,'youtube-downloads')
 
+# Create a service object for interacting with the API
+youtube = build('youtube', 'v3', developerKey = "GOOGLE_DEV_API_KEY")
```

Comment:
hier bitte das folgende tun:

in settings.py den GOOGLE_DEV_API_KEY laden aus dem ENV und wenn er nicht drin ist dann schreib da eben sowas wie "GOOGLE_DEV_API_KEY" rein als alterntive (mit os.environ.get und so ihr wisst schon)

Reply:
dann hier aus settings einfach importieren